app/setup_handlers.go

```diff
@@ -1,34 +1,18 @@
 package app
 
 import (
-	"github.com/Sifchain/sifnode/app/ante"
 	storetypes "github.com/cosmos/cosmos-sdk/store/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	m "github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/cosmos/cosmos-sdk/x/authz"
 	"github.com/cosmos/cosmos-sdk/x/upgrade/types"
 )
 
-const releaseVersion = "0.15.0"
+const releaseVersion = "0.16.0"
```

Comment:
@Jedi2002 I would suggest to first set a RC version such as `0.16.0-rc.2` (as we already have `0.16.0-rc.1` published with the latest PR merged to master) until we completed all the regression tests before cut the official release 0.16.0

Reply:
okay